DataAccess/DataAccess.Test/AbstractRepositoryContainerTest.cpp

```diff
@@ -0,0 +1,29 @@
+#include <catch2/catch.hpp>
+
+#include "DataAccess/AbstractRepositoryContainer.hpp"
+#include "DataAccess.Postgre/PostgreAdapter.hpp"
+#include "DataAccess.Postgre/PostgreRepositories.hpp"
+
+using DataAccess::PostgreAdapter;
+
+TEST_CASE("AbstractRepositoryContainer test","[dummy]")
+{
+	DataAccess::AbstractRepositoryContainer testContainer(PostgreAdapter::Instance());
```

Comment:
Do you cover constructor by tests?

Reply:
Sonar want's it

Reply:
In the 'dev' branch it isn`t covered
![image](https://user-images.githubusercontent.com/90044672/175992858-9538845b-3b60-4362-bdb2-e2e07b93aaac.png)